tv_apps/smart-system/js/app_transition_controller.js

```diff
@@ -1,7 +1,12 @@
-/* global AppWindowManager, SettingsCache, Service */
+/* global AppWindowManager, SettingsCache, Service, Animations,
+          homescreenWindowManager */
 'use strict';
 
 (function(exports) {
+
+  const GRAY_BACKGROUND_APPS
```

Comment:
For the line breaking, I would suggest the following way:
```
const GRAY_BACKGROUND_APPS = [
  'app://app-deck.gaiamobile.org/manifest.webapp'
];
```
This is because we may add others apps into gray background apps in the future. It is easy to add another one in this coding style.